hw3/src/pages/population.jsx

```diff
@@ -1,11 +1,87 @@
 import Navbar from "../components/navbar";
+import { useEffect, useState } from "react";
+import { Bar } from "react-chartjs-2";
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend,
+} from "chart.js";
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend
+);
 
 const Header = () => {
   return (
     <div className="text-center mt-10">
-      <p className="text-5xl font-bold font-sans text-blue-600">
-        The Population of contries in South America
-      </p>
+      <h1 className="text-5xl font-bold font-sans text-blue-600">
+        The Population of Countries in South America
+      </h1>
+    </div>
+  );
+};
+
+const GetPopulation = () => {
+  const [chartData, setChartData] = useState({
+    labels: [],
+    datasets: [
+      {
+        label: "Population",
+        data: [],
+        backgroundColor: "rgba(75, 192, 192, 0.6)",
+        borderColor: "rgba(75, 192, 192, 1)",
+        borderWidth: 1,
+      },
+    ],
+  });
+
+  useEffect(() => {
+    const fetchCountries = async () => {
+      try {
+        const res = await fetch(
+          "https://cs464p564-frontend-api.vercel.app/api/countries"
+        );
+        const data = await res.json();
```

Comment:
The fetched data, won't it be better to not fetched them all over the pages. For example, in your list, you fetched the data using a function, and this page also fetches the data.